jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -57,10 +61,17 @@ class UnitConversion(PluginTemplateMixin):
     flux_unit_items = List().tag(sync=True)
     flux_unit_selected = Unicode().tag(sync=True)
 
-    show_translator = Bool(False).tag(sync=True)
+    sb_unit_items = List().tag(sync=True)
+    sb_unit_selected = Unicode().tag(sync=True)
+
     flux_or_sb_items = List().tag(sync=True)
     flux_or_sb_selected = Unicode().tag(sync=True)
 
+    # in certain configs, a pixel scale factor will not be in the FITS header
+    # we need to disable translation in the API and UI variables/functions.
+    flux_or_sb_config_disabler = Unicode().tag(sync=True)
```

Comment:
this is probably ok for now since its private.  I think in the future, as we also add velocity display units, etc, it might make sense to have a traitlet that contains a list of display units to expose based on the config and loaded data, but that might be overengineering in the meantime.

Reply:
I can imagine the infrastructure for this, I do think there need to be some determination of whether to display flux, sb, or both depending on the config and whatever the unit of the data item is (which could be done with your original comment about using the config and flux_or_sb). Once that's determined you should be able to tell which dropdown(s) and API should show up, and then have a function or something to populate the units for that traitlet (it might need to be 2?).

Reply:
ok, let's just keep this in mind as we move forward, but I'm ok with it as-is for now since its not exposed.